### ISSUE TITLE
ERROR:root:Subdemonination conversion impossible 1 gwei

### ISSUE THREAD

**Issue by kaleofduty** [crytic/slither#772]
The following contract `Sample.sol` causes slither to exit with an error.

```
// SPDX-License-Identifier: MIT
pragma solidity ^0.7.0;

contract Sample {
  uint256 internal variable;
  function foo() external {
    variable = 1 gwei;
  }
}
```

```bash
$ slither Sample.sol         
[...]                 
slither.exceptions.SlitherException: Subdemonination conversion impossible 1 gwei
ERROR:root:Error:
ERROR:root:Subdemonination conversion impossible 1 gwei
ERROR:root:Please report an issue to https://github.com/crytic/slither/issues
```

Solidity has no problems compiling it:
```bash
$ solc Sample.sol
Compiler run successful, no output requested.
```

Versions:
```bash
$ python --version
Python 3.8.5
$ pip freeze
crytic-compile==0.1.12
prettytable==2.0.0
pysha3==1.0.2
slither-analyzer==0.7.0
wcwidth==0.2.5
```

**Comment by montyly** [comment]
Hi @kaleofduty. Thank you for reporting this. We will add support for the missing subdemonination.

Related #748

**Comment by montyly** [comment]
Hi @kaleofduty 

We added the support in `master`, the fix will be available in the upcoming [0.7.1](https://github.com/crytic/slither/issues/789) release
